src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String[] entries = getTableEntries(fromFileName);
+        if (entries.length == 0 || entries[0].isEmpty()) {
+            return;
+        }
+        String[] reportData = getReportData(entries);
+        clearFile(toFileName);
+        for (String entry : reportData) {
+            try (BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(toFileName,
```

Comment:
The `FileWriter` is opened in append mode (`true`), which may not be necessary since the file is cleared before writing. Consider using `false` to overwrite the file content if the file is always cleared before writing.